File: #0168.excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {

        string result; 
        while (columnNumber) {
            result.insert(result.begin(), 'A' + (columnNumber - 1) % 26);
            columnNumber = (columnNumber - 1) / 26;
        }
        return result;
    }
};

File: #0168.excel-sheet-column-title.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function(columnNumber) {
    let result = '';
    let title = columnNumber;
    while (title != 0) {
        let rem = title % 26 ? title % 26 : 26;
        title = (title - rem) / 26;
        result = String.fromCharCode(rem + 64) + result;
    }
    return result;
};
